source/_integrations/electricity_maps.markdown

```diff
@@ -0,0 +1,28 @@
+---
```

Comment:
Should I rename this file back to `co2signal.markdown` and just change content, or is this okay with the redirect?

Reply:
You can do both, so old links do not break. You can search for references to that can be replaced.

Reply:
That is incorrect. The file need to match the domain of the integration.

Reply:
Okay thanks, is there no way we can show a `/electricity_maps` URL instead of `/co2signal` without changing the domain? :)

Reply:
The domain key will be around forever, unless you have a way to migrate it in core so it won't break any system.

For example, <https://www.home-assistant.io/integration/hassio>, which is the documentation for our own Supervisor application. `hassio` hasn't been used as a term since years anymore 🤷

Reply:
Thanks for the clarification, I have renamed the file and removed the redirect 👍